Sources/GPGXBridge/GPGXEmulatorBridge.m

```diff
@@ -131,22 +131,22 @@ - (void)runFrameAndProcessVideo:(BOOL)processVideo
 
 #pragma mark - Inputs -
 
-- (void)activateInput:(NSInteger)inputValue value:(double)value
+- (void)activateInput:(NSInteger)inputValue value:(double)value at:(NSInteger)playerIndex
 {
-    int player = 0;
-    input.pad[player * 4] |= inputValue;
+    input.pad[playerIndex * 4] |= inputValue;
```

Comment:
Why is this `playerIndex * 4`?

Reply:
I'm not sure tbh. The old `player` var appeared to simulate the purpose of the `playerIndex`, so I swapped it out for the new player index and tested local multiplayer, and IIRC things appeared to work correctly, so I left it in 🤷‍♂️